src/test/suite/mdbExtensionController.test.ts

```diff
@@ -700,31 +690,21 @@ suite('MDBExtensionController Test Suite', function () {
       mockActiveConnectionId
     );
 
-    const mockOpenTextDocument = sinon.fake.resolves('untitled');
-    sinon.replace(vscode.workspace, 'openTextDocument', mockOpenTextDocument);
+    await vscode.commands.executeCommand('mdb.addCollection', mockTreeItem);
 
-    const mockShowTextDocument = sinon.fake();
-    sinon.replace(vscode.window, 'showTextDocument', mockShowTextDocument);
+    const activeTextEditor =
+      mdbTestExtension.testExtensionController._playgroundController
+        ._activeTextEditor;
+    assert.strictEqual(activeTextEditor?.document.languageId, 'javascript');
+    assert(activeTextEditor?.document.uri.scheme === 'untitled');
 
-    await vscode.commands.executeCommand('mdb.addCollection', mockTreeItem);
+    const content = activeTextEditor.document.getText();
+    assert(content.includes('// The current database to use.'));
+    assert(content.includes('iceCreamDB'));
+    assert(content.includes('NEW_COLLECTION_NAME'));
+    assert(!content.includes('time-series'));
 
-    assert(mockOpenTextDocument.firstCall.args[0].language === 'mongodb');
-    assert(
-      mockOpenTextDocument.firstCall.args[0].content.includes(
-        '// The current database to use.'
-      )
-    );
-    assert(
-      mockOpenTextDocument.firstCall.args[0].content.includes('iceCreamDB')
-    );
-    assert(
-      mockOpenTextDocument.firstCall.args[0].content.includes(
-        'NEW_COLLECTION_NAME'
-      )
-    );
-    assert(
-      !mockOpenTextDocument.firstCall.args[0].content.includes('time-series')
-    );
+    await vscode.commands.executeCommand('workbench.action.closeActiveEditor');
```

Comment:
Is this test cleanup? It might be better to be in a `beforeEach` so that if this test fails before reaching this line it doesn't impact other tests. Same with other occurrences in this file.

Reply:
Yeah left over last commit refactoring. Will remove! Thx for noticing!